apps/server/src/modules/learnroom/learnroom.module.ts

```diff
@@ -37,7 +38,7 @@ import { RoomsService } from './uc/rooms.service';
 import { RoomsUc } from './uc/rooms.uc';
 
 @Module({
-	imports: [AuthorizationModule, FilesStorageClientModule],
+	imports: [AuthorizationModule, FilesStorageClientModule, UserModule],
```

Comment:
UserModule is only needed for the PermissionService? Than it should be removed. Instead use the AuthorizationService from the AuthorizationModule

Reply:
Done.